cmd/cronjob_receive_adapter/main.go

```diff
@@ -37,6 +37,9 @@ const (
 
 	// Sink for messages.
 	envSinkURI = "SINK_URI"
+
```

Comment:
Please migrate this to envconfig, an example https://github.com/botless/commands/blob/master/cmd/core/main.go#L16

Reply:
Done! Thanks for the quick review. I think the lgtm is gone as there were new changes...